app/javascript/react/components/Graph/chartEvents.ts

```diff
@@ -2,77 +2,55 @@ import Highcharts from "highcharts/highstock";
 import graphChevronLeft from "../../assets/icons/graphChevronLeft.svg";
 import graphChevronRight from "../../assets/icons/graphChevronRight.svg";
 
-const addNavigationArrows = (chart: Highcharts.Chart, setTooltipVisible: React.Dispatch<React.SetStateAction<boolean>>) => {
-  // Remove existing arrows if any
-  chart.renderer.boxWrapper.element.querySelectorAll('.custom-arrow').forEach(el => el.remove());
-
+const addNavigationArrows = (chart: Highcharts.Chart) => {
   const chartWidth = chart.chartWidth;
   const chartHeight = chart.chartHeight;
 
+  // Remove existing arrows if any
+  chart.renderer.boxWrapper.element
+    .querySelectorAll('.custom-arrow')
+    .forEach((el) => el.remove());
+
   const leftArrow = chart.renderer
     .image(graphChevronLeft, 30, chartHeight / 2, 30, 30)
     .attr({ zIndex: 10, class: 'custom-arrow' })
+    .css({ cursor: 'pointer' })
     .add();
+
   const rightArrow = chart.renderer
     .image(graphChevronRight, chartWidth - 80, chartHeight / 2, 30, 30)
     .attr({ zIndex: 10, class: 'custom-arrow' })
+    .css({ cursor: 'pointer' })
     .add();
 
-  const moveLeft = () => {
-    const axis = chart.xAxis[0];
-    const { min, max, dataMin, dataMax } = axis.getExtremes();
-    const range = max - min;
-    const newMin = Math.max(dataMin, min - range * 0.1);
-    const newMax = Math.max(dataMin + range, max - range * 0.1);
-    axis.setExtremes(newMin, newMax);
-    updateArrowState();
-  };
-
-  const moveRight = () => {
+  const move = (direction: 'left' | 'right') => {
```

Comment:
can this direction also be a type? When we use a string here and then again below it is more error-prone. Having it and using constant could help